app/Http/Controllers/HomeController.php

```diff
@@ -111,23 +116,25 @@ public function search(SearchRequest $request)
         return view('search', $data);
     }
 
-    public function getPhone($phone)
+    public function getPhone($phone, $radio)
     {
         $pattern = '/^(\+84|0)\d{9,10}$/';
         if (preg_match($pattern, $phone)) {
             $id = Auth::id();
             $data = [
                 'phone' => $phone,
             ];
+            $this->usermeta->updateDisplayPhone($id, $radio);
             $this->user->update($id, $data);
 
+
             return response()->json([
                 'data' => '1',
             ]);
         } else {
             return response()->json([
                 'data' => '0',
             ]);
-        }
+        }        
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Http/Controllers/HomeController.php</br>**Line**: 138</br>Whitespace found at end of line